src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -20,8 +34,15 @@ class Focuser(AbstractSerialFocuser):
     def __init__(self,
                  name='Astromechanics Focuser Controller',
                  model='astromechanics',
+                 initial_position=0,
+                 dev_node_pattern='/dev/tty.usbserial-AG0*',
+                 serial_number_pattern=serial_number_pattern,
```

Comment:
The dev node pattern should be `None` in POCS to keep it generic and the values can come from the config.

But, if you are not doing anything with these parameters here and they are already in the super class, you don't really need to specify them, they will just go through with `**kwargs`

```suggestion
                 dev_node_pattern=None,
                 serial_number_pattern=serial_number_pattern,
```